q4.js

```diff
@@ -0,0 +1,37 @@
+let sa = function SortArray(inputArray){
+    this.originalArray = inputArray;
+    this.getSortedArray = function(inArray){
+      return arraySort(inArray);
+    }
+    let arraySort = function(arr){
+      arr.sort(function(a,b){return a-b;});
+      return arr;
+    }
+  }
+  
+  let inArray = [12,34,-5,3,24];
+  console.log(inArray);
+  let sarr = new sa(inArray);
+  console.log(sarr);
+  let newSorted = sarr.getSortedArray(inArray);
```

Comment:
instead of passing the array as an argument here, you should use this.originalArray in the getSortedArray() this is because the array is already passed while creating the object.

Reply:
Made the changes accordingly